### ISSUE TITLE
Translation Guide

### ISSUE THREAD

**Issue by Shoalsteed** [Shoalsteed/UX#41]
Change the priority to join tx https://www.transifex.com/otf/I2P/

- how many people use the old way?
- if we keep those instructions, need to update them for gitlab.
- "Come to #i2p-dev on irc and talk to people" & "Yes, we know it is somewhat of a hurdle to get started. It's really the only possible way we can do it. Give it a try, it really isn't that hard." - these are not welcoming or encouraging ways to get people involved. It is making people come to the project and telling it is not that hard, and that can be a bit off-putting in tone.

**Comment by Shoalsteed** [comment]
Translation of the website and router console is done with .po files. The easiest way by far to translate the website is to sign up for an account at Transifex and request to join a translation team. Translation of the website is done with .po files. Alternatively it can be done as outlined below.

On the I2P Transifex page, you will see all the languages that people are working on for translations and their progress. Once you sign up for a Transifex account, you can work on the localization effort of your choice.

**Comment by Shoalsteed** [comment]
**REMOVE**

_Preparation
Come to #i2p-dev on irc and talk to people. Claim the language - To make sure other coworkers don't bump onto the files you are working on, please update the translation status on this wiki page.
Follow the new developer's guide, Including the installation of monotone, checking out i2p.www branch, and generate your own monotone keys. It is not required that you sign a dev agreement._

**Comment by Shoalsteed** [comment]
**REMOVE** 

_As you can see, it's not that difficult. If you have questions about the meaning of the terms in the console, ask in #i2p-dev on IRC._

**Comment by Shoalsteed** [comment]
***** it may be worth it to direct people to Discord rather than irc.

**Comment by Shoalsteed** [comment]
**THIS NEEDS TO BE ENTIRELY REWORKED** 

FAQ
Q: Why do I have to install monotone, Java, jsp, learn about .po files and html, etc.? Why can't I just do a translation and email it to you?

A: Several reasons:

You might be interested in translating via Transifex. Request to join a translation team here.
We don't have anybody who has time to accept manual contributions and submit them to our source control system on your behalf. Even if we did, it doesn't scale.
Maybe you are thinking translation is a one-step process. It isn't. You can't do it all at once. You will make mistakes. You need to test it and tweak it to make it look right before you submit it. Developers will update or add to the English text, thus requiring a translation update.
Having translators use a source control system directly provides authentication and accountablility - we know who is doing what, and we can track changes, and revert them if necessary.
.po files are not difficult. If you don't want to work directly with them, we recommend 'poedit'.
HTML files are not difficult. Just ignore the html stuff and translate the text.
Installing and using monotone is not that difficult. Several of the translators and other contributors to I2P are non-programmers, and they use monotone regularly. Monotone is simply a source control system, it is not about "coding".
Our items to translate are not "documents". They are html files and po files, with a specific format and character encoding (UTF-8) that must be maintained, and not corrupted by email programs or other methods of transfer.
We looked at 'pootle' as a front-end for translators. It didn't work well, needed an administrator, and a pootle-based process would suffer from a number of the above flaws.
REMOVE THIS 
In summary: Yes, we know it is somewhat of a hurdle to get started. It's really the only possible way we can do it. Give it a try, it really isn't that hard.
